### ISSUE TITLE
let's create a new command utils

### ISSUE THREAD

**Issue by mikkokotila** [autonomio/autonomio#47]
Under autonomio.commands, let's create a utils() function where we have two different utils for now: 

- utils.vectorize()  # this is from vectorize_text.py
- utils.ascify() # this is from ascify.py

**Comment by mikkokotila** [comment]
Before we can do this, we have to streamline the way new commands are added, and how parameters are added to commands that are already there. Now the cognitive load required to do it is high (and it really should not be). Will create a new issue for that.

**Comment by mikkokotila** [comment]
This is related with #62
